test/integration/ComputeFlow.test.ts

```diff
@@ -940,15 +940,18 @@ describe('Compute flow', () => {
     const algoDefinition: ComputeAlgorithm = {
       meta: algorithmMeta
     }
-    const order = await ocean.compute.orderAsset(
-      bob.getId(),
-      datasetNoRawAlgo.id,
-      service1.index,
-      algoDefinition,
-      null, // no marketplace fee
-      computeAddress // CtD is the consumer of the dataset
-    )
-    assert(order === null, 'Order should be null')
+    try {
+      await ocean.compute.orderAsset(
+        bob.getId(),
+        datasetNoRawAlgo.id,
+        service1.index,
+        algoDefinition,
+        null, // no marketplace fee
+        computeAddress // CtD is the consumer of the dataset
+      )
+    } catch (error) {
+      assert(error != null, 'Order should be fail.')
```

Comment:
```suggestion
      assert(error != null, 'Order should not fail.')
```

Reply:
I think the message should still indicate that the order `'Order should fail.'` in this test case, but as with the test in the MarketplaceFlow i will add inside try block an assert to catch if the order succeeds and have an assertion failed.